junit5-engine/src/main/java/org/junit/gen5/engine/junit5/execution/ConditionEvaluator.java

```diff
@@ -108,4 +137,42 @@ private void logResult(Class<?> conditionType, ConditionEvaluationResult result)
 		LOG.finer(() -> String.format("Evaluation of condition [%s] resulted in: %s", conditionType.getName(), result));
 	}
 
+	private Predicate<Object> conditionIsActivated(ConfigurationParameters configurationParameters) {
+		Pattern deactivatePattern = getDeactivatePattern(configurationParameters);
+		if (deactivatePattern != null) {
+			if (DEACTIVATE_ALL_CONDITIONS_PATTERN.equals(deactivatePattern)) {
+				return alwaysDeactivated;
+			}
+			return condition -> !deactivatePattern.matcher(condition.getClass().getName()).matches();
+		}
+		return alwaysActivated;
+	}
+
+	private Pattern getDeactivatePattern(ConfigurationParameters configurationParameters) {
+		// @formatter:off
+		return configurationParameters.get(DEACTIVATE_CONDITIONS_PATTERN_PROPERTY_NAME)
+				.filter(StringUtils::isNotBlank)
+				.map(this::convertToRegEx)
+				.map(Pattern::compile)
+				.orElse(null);
+		// @formatter:on
+	}
+
+	/**
+	 * See {@link Constants#DEACTIVATE_CONDITIONS_PATTERN_PROPERTY_NAME} for
+	 * details on the pattern matching syntax.
+	 */
+	private String convertToRegEx(String pattern) {
+		pattern = Matcher.quoteReplacement(pattern.trim());
+
+		// Match "." against "." and "$" since users may declare a "." instead of a
+		// "$" as the separator between classes and nested classes.
+		pattern = pattern.replace(".", "[.$]");
+
+		// Convert our "*" wildcard into a proper RegEx pattern.
+		pattern = pattern.replace("*", ".+");
```

Comment:
According to the documentation an asterisk "will match against zero or more characters in the FQCN" while `.+` requires at least one character.